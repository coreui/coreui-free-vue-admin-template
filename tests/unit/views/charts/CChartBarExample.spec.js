import Vue from 'vue'
import { shallowMount } from '@vue/test-utils';
import CoreuiVue from '@coreui/vue'
import CChartBarExample from '@/views/charts/CChartBarExample'

Vue.use(CoreuiVue)

describe('CChartBarExample', () => {
  it('has a name', () => {
    expect(CChartBarExample.name).toBe('CChartBarExample')
  })
  it('is Vue instance', () => {
    const wrapper = shallowMount(CChartBarExample)
    expect(wrapper.isVueInstance()).toBe(true)
  })
  it('is CChartBarExample', () => {
    const wrapper = shallowMount(CChartBarExample)
    expect(wrapper.is(CChartBarExample)).toBe(true)
  })
  test('renders correctly', () => {
    const wrapper = shallowMount(CChartBarExample)
    expect(wrapper.element).toMatchSnapshot()
  })
  it('should have computed', () => {
    const wrapper = shallowMount(CChartBarExample)

    expect(wrapper.vm.defaultDatasets).toBeDefined()
  })
})
