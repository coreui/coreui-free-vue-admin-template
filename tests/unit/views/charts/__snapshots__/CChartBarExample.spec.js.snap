// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CChartBarExample renders correctly 1`] = `
<cchartbar-stub
  datasets="[object Object]"
  labels="months"
/>
`;
